### ISSUE TITLE
False-positive `::picker(select)` and `::picker-icon` errors

### ISSUE THREAD

**Issue by tyrann0us** [stylelint/stylelint#8622]
### What minimal example or steps are needed to reproduce the bug?

```
::picker(select) {
  color: red;
}

select::picker-icon {
  color: red;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "selector-pseudo-element-no-unknown": true
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcD4AcCWYDWMBOAKAzjADYxgAuAlAATAA6AdlVZERNvFdjFANwMC+DBoRLl4KdFmwBadBEZ1GzCK3adufeoPogANCABmqEgDkAhgFs4iGAA9LyEgDow+fHvDyjAcwQhFTLQgdmQw9FD4QRwA2gxMgSD4ZACeJESo9GTSkPQ+0klm4WbYUEFxVAC6Ah45PgBibBZmZH4AVvjyHrDI7ogBVEFJqcQZZFEDIETNMElBuuWDKWmj2V6o3vlkhVDFpQgTU6GzINX8QA)

### Which Stylelint-related dependencies are you using?

Stylelint wrapped in [`@wordpress/scripts`](https://www.npmjs.com/package/@wordpress/scripts) `v30.15.0`.

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problems were reported:

```
example.css
1:1-9  ✖ Unexpected unknown pseudo-element selector "::picker"      (selector-pseudo-element-no-unknown)
5:7-20 ✖ Unexpected unknown pseudo-element selector "::picker-icon" (selector-pseudo-element-no-unknown)

### Do you have a proposal to fix the bug?

Add both names to Stylelint.
